__tests__/app.test.js

```diff
@@ -243,4 +244,93 @@ describe('app.js', () => {
         });
     });
   });
+
+  describe('8. PATCH /api/reviews/:review_id', () => {
+    test('returns the updated review', () => {
+      const votesToUpdate = {
+        inc_votes: 1000,
+      };
+      return request(app)
+        .patch('/api/review/3')
+        .send(votesToUpdate)
+        .expect(202)
+        .then((response) => {
+          expect(response.body).toEqual({
+            updatedReview: [
+              {
+                category: 'social deduction',
+                created_at: '2021-01-18T10:01:41.251Z',
+                designer: 'Akihisa Okui',
+                owner: 'bainesface',
+                review_body: "We couldn't find the werewolf!",
+                review_id: 3,
+                review_img_url:
+                  'https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?w=700&h=700',
+                title: 'Ultimate Werewolf',
+                votes: 1005,
+              },
+            ],
+          });
+        });
+    });
+    test('returns a status 400 when passed an invalid query', () => {
+      const votesToUpdate = {
+        inc_votes: 1000,
+      };
+      return request(app)
+        .patch('/api/review/banana')
+        .send(votesToUpdate)
+        .expect(400);
+    });
+    test('returns a status 404 when passed a review ID that does not exist', () => {
+      const votesToUpdate = {
+        inc_votes: 1000,
+      };
+      return request(app)
+        .patch('/api/review/999')
+        .send(votesToUpdate)
+        .expect(404);
+    });
+    test('returns a status 400 when passed an invalid body', () => {
+      const votesToUpdate = {
+        random: 1000,
+      };
+      return request(app)
+        .patch('/api/review/3')
+        .send(votesToUpdate)
+        .expect(400);
+    });
+  });
+  describe('9. GET /api/users', () => {
+    test('returns an array of objects', () => {
+      return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then((response) => {
+          console.log(response.body);
+          expect(Array.isArray(response.body)).toBe(true);
+          expect(typeof response.body[0]).toBe('object');
```

Comment:
I'd suggest iterating over every object for this test, rather than checking only one element